roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -113,7 +113,7 @@
     {% if kubeadm_version is version('v1.14.0', '>=') %}
     {% if kubeadm_version is version('v1.15.0', '<') %}
     --experimental-upload-certs
-    {% if kubeadm_certificate_key is defined %}
+    {% if kubeadm_certificate_key is defined and kubeadm_version is version('v1.15.0', '<') %}
```

Comment:
Same here

Reply:
This change is not needed, there is already `{% if kubeadm_version is version('v1.15.0', '<') %}` on line 114.